ci/Dockerfile

```diff
@@ -98,9 +100,6 @@ RUN set -e; \
     && sdk install maven $MAVEN_VERSION \
     && sdk install gradle $GRADLE_VERSION \
     && sdk install sbt $SBT_VERSION \
-    && sdk offline enable \
```

Comment:
These are breaking changes that also alter the threat model, since we are essentially unlocking and making the sdk command available for cdxgen and other tools.